### PR TITLE
Cleanup tests

### PR DESCRIPTION
**Pull request by fvovan**
- Use `assert` in all tests to be consistent. Including the one that are not merged. FYI @tomaszdurka.
- Rename jobManager => job-manager. Delete all obsolete `require` in it.

### CODE REVIEW

**Comment by fvovan** [comment]
@vogdb please review,
https://github.com/cargomedia/cm-janus/pull/45 and 
https://github.com/cargomedia/cm-janus/pull/43 cleanings are also done.

**Comment by vogdb** [comment]
looks good. One thing in #45 `serviceLocator.register('logger', new Logger());` I guess should be changed too.

File: test/spec/job-manager.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ var JobManager = require('../../lib/job/job-manager');
 var AbstractJobHandler = require('../../lib/job/handler/abstract');
 var Logger = require('../../lib/logger');
 var serviceLocator = require('../../lib/service-locator');
-serviceLocator.register('logger', new Logger());
+serviceLocator.register('logger', function() {
+  return new Logger();
+});
```
Comment (tomaszdurka) [inline test/spec/job-manager.js]: You don't need to make it lazy.

Comment (tomaszdurka) [inline test/spec/job-manager.js]: Matter of preference I guess.

Comment (fvovan) [inline test/spec/job-manager.js]: Not completely,
Mocha runs tests in a parallel, so you have the same ServiceLocator per each test.
And if you register service by value (not by  function) then it's impossible to override it with function, as it checks if value is defined first and then calls function. Can be changed later but just non dirty fix for now

**Pull request merged**